agave_app/agaveGui.cpp

```diff
@@ -121,9 +122,13 @@ agaveGui::createActions()
   m_toggleCameraProjectionAction->setStatusTip(tr("Toggle perspective and orthographic camera projection modes"));
   connect(m_toggleCameraProjectionAction, SIGNAL(triggered()), this, SLOT(view_toggleProjection()));
 
-  m_saveImageAction = new QAction(tr("&Save image..."), this);
+  m_saveImageAction = new QAction(tr("&Quick render..."), this);

```

Comment:
I agree, it will probably be renamed again after some UX review.  It could also be named "Capture Viewport" or "Save current view" to distinguish it from "launch render dialog where I am going to take time to set up further image generation parameters"